ImageFeed/Helpers/UIColor + Extensions.swift

```diff
@@ -12,3 +12,65 @@ extension UIColor {
     static var ypWhite: UIColor { UIColor(named: "white") ?? UIColor.white }
     static var ypGray: UIColor { UIColor(named: "gray") ?? UIColor.gray }
 }
+
+extension URLSession {
+    private enum FetchError: Error {
+        case codeError
+    }
+
+    func objectTask<T: Decodable>(
+        for request: URLRequest,
+        completition: @escaping (Result<T, Error>) -> Void
+    ) -> URLSessionTask {
+        let task = dataTask(with: request) {
+            data, response, error in
+            DispatchQueue.global(qos: .utility).async {
+                if
+                    let error {
+                    completition(.failure(error))
+                }
```

Comment:
если в 120 символов строка умещается, и нет многострочных выражений, нет смысла делать перенос
```suggestion
                if let error {
                    completition(.failure(error))
                }
```

Reply:
fixed